project/plugins.sbt

```diff
@@ -4,6 +4,7 @@ addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.3")
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.7")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
 addSbtPlugin("org.scalastyle" % "scalastyle-sbt-plugin" % "1.0.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.0")
```

Comment:
hey, you should be able to remove this plugin after applying the rules! We should add that to the Scio migration guide...